### ISSUE TITLE
allman style brace formatting triggers suspicious_else_formatting

### ISSUE THREAD

**Issue by wareya** [rust-lang/rust-clippy#3864]
.

**Comment by flip1995** [comment]
Example:
```rust
fn main() {
    if true 
    {
        println!("Hi");
    }
    else 
    {
        println!("Bye");
    }
}
```

[Playground](https://play.rust-lang.org/?version=nightly&mode=debug&edition=2018&gist=bfecd4e8c7f6a64fd2bb59828d691658)

**Comment by sdegutis** [comment]
I just switched to this style and now all my else-statements are being triggered. Is there at least a way to tell clippy to silence this warning for the whole project? I can't find what to put inside .clippy.toml to silence this, if any such thing exists.

**Comment by flip1995** [comment]
clippy.toml doesn't and won't support allowing/denying lints. You can do this in the `main.rs`/`lib.rs` file with `#![allow(clippy::suspicious_else_formatting)]` at the top of the file.

**Comment by chilabot** [comment]
I prefer to put linting in the Cargo.toml file, in the `lints.clippy` section. More accessible and visually better.
